includes/Deactivation.php

```diff
@@ -30,9 +30,54 @@ class Deactivation {
 	public function __construct( Container $container ) {
 		$this->container = $container;
 
+		register_deactivation_hook(
+			$container->plugin()->file,
+			array( $this, 'handle' )
+		);
+
 		// Plugin deactivation survey.
 		add_action( 'admin_head-plugins.php', function () {
 			new DeactivationSurvey();
 		} );
 	}
+
+	/**
+	 * Handle deactivation.
+	 *
+	 * @return void
+	 */
+	public function handle() {
+		/*
+		 * The data module is currently handing the 'site_launched' event.
+		 * We are commenting out the code below but keeping it as example for future events.
+		 */
+		// $this->site_launched_event();
```

Comment:
@wpalani I'm guessing that if this is commented out we can remove this line and the related method?

Reply:
This has been fixed in a recent commit.